README.md

```diff
@@ -10,117 +10,140 @@
+**Default Title Regex:** `^(?i:release)` (title must start with the word "release" (case insensitive))
 
 **Default Version Number Regex:** This Regex will be checked against a Pull Request title.
-This follows [``SemVer``](https://regex101.com/r/Ly7O1x/3/) (Semantic Versioning) pattern.
-e.g. ``1.0.0``, ``1.0``, ``v1.0.1`` etc.
+This follows [`SemVer`](https://regex101.com/r/Ly7O1x/3/) (Semantic Versioning) pattern.
+e.g. `1.0.0`, `1.0`, `v1.0.1` etc.
 
 For more details on **Semantic Versioning pattern** go to this link: https://regex101.com/r/Ly7O1x/3/
 
-**Note:** [you can provide your own ``regex`` through the ``config`` file](#config-file-usage-optional)
+**Note:** You can provide your own `regex` using [the `config` file](#config-file-usage)
 
-To **Enable Commenting, Disable Committing, Group changelog items** and some other options
-Look at the [config file docs](#config-file-usage-optional).
+To **Enable Commenting, Disable Committing, Group changelog items** and  
+some other options, see [config file usage (optional)](#config-file-usage-optional).
 
-To integrate ``Changelog CI`` with your repositories Actions,
-Put this step inside your ``.github/workflows/workflow.yml`` file:
+To integrate `Changelog CI` with your repositories Actions,
+Put this step inside your `.github/workflows/workflow.yml` file:
 
 ```yaml
 - name: Run Changelog CI
     uses: saadmk11/changelog-ci@v0.7.0
     with:
       # Optional, you can provide any name for your changelog file,
-      # defaults to ``CHANGELOG.md`` if not provided.
+      # defaults to `CHANGELOG.md` if not provided.
       changelog_filename: MY_CHANGELOG.md
       # Optional, only required when you want more customization
       # e.g: group your changelog by labels with custom titles,
       # different version prefix, pull request title and version number regex etc.
       config_file: changelog-ci-config.json
       # Optional, This will be used to configure git
-      # defaults to ``github-actions[bot]`` if not provided.
+      # defaults to `github-actions[bot]` if not provided.
       committer_username:  'test'
       committer_email:  'test@test.com'
     env:
-      # optional, only required for ``private`` repositories
-      # and required if the changelog comment 
+      # optional, only required for `private` repositories
+      # and required if the changelog comment
       # option is turned on through the config file
       GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 ```
 
 **Changelog CI Badge:**
 
 ```markdown
-![Changelog CI Status](https://github.com/<username>/<repo_name>/workflows/Changelog%20CI/badge.svg)
-``` 
+![Changelog CI Status](https://github.com/<username>/<repository>/workflows/Changelog%20CI/badge.svg)
+```
 
 **Output:**
 
 ![Changelog CI Status](https://github.com/saadmk11/changelog-ci/workflows/Changelog%20CI/badge.svg)
 
 
-## Config File Usage (Optional):
+## Configuration
```

Comment:
I think we should keep the ` (Optional):`

Reply:
I agree that configurations themselves should be optional, but I think users will find this page easier to read if we stick to the common convention. I made this change after consulting `man(1) man`, what I consider to be a somewhat canonical source of truth for writing documentation:

>  A manual page consists of several sections.
> 
> Conventional  section  names  include **NAME**, **SYNOPSIS**, **CONFIGURATION**, **DESCRIPTION**, **OPTIONS** ... **NOTES**, **BUGS**, **EXAMPLE**, **AUTHORS**, and **SEE ALSO**.

Perhaps a compromise: we emphasize that configurations are ***optional*** directly below the `Configuration` heading

Reply:
I still feel like it's more straightforward to put it in the header. It will be easy to find and create less confusion :)

Reply:
I see where you're coming from. Fixed in https://github.com/saadmk11/changelog-ci/pull/46/commits/a00cd665a8dbfea2bd12149ee7c227181d66caf9

Reply:
Looks great, Thanks 💯